tests/create_survey_dcip_test.py

```diff
@@ -0,0 +1,82 @@
+#  Copyright (c) 2021 Mira Geoscience Ltd.
+#
+#  This file is part of geoh5py.
+#
+#  geoh5py is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  geoh5py is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public License
+#  along with geoh5py.  If not, see <https://www.gnu.org/licenses/>.
+
+# pylint: disable=R0914
+
+import tempfile
+from pathlib import Path
+
+import numpy as np
+
+from geoh5py.objects import CurrentElectrode, PotentialElectrode
+from geoh5py.shared.utils import compare_entities
+from geoh5py.workspace import Workspace
+
+
+def test_survey_dcip():
+
+    name = "TestCurrents"
+    n_data = 12
+
+    with tempfile.TemporaryDirectory() as tempdir:
```

Comment:
Could use tmp_path fixture here, obviously not required though..

Reply:
I ll do it for them all at once on a separate issue?